surfer/tests/test_viz.py

```diff
@@ -113,21 +113,22 @@ def test_brains():
 @requires_fsaverage
 def test_annot():
     """Test plotting of annot."""
-    mlab.options.backend = 'test'
+    mlab.options.backend = 'auto'
```

Comment:
Argh. Okay. If the CIs are okay with it I can live with it.

BTW I just pushed a fix to `master` that I made a few days ago to deal with the `imwrite` failure. I'll restart CIs on your latest commit so they hopefully pick it up.